common/constant/key.go

```diff
@@ -389,14 +389,14 @@ const (
 
 // reflection service
 const (
-	ReflectionServiceTypeName  = "DubbogoServerReflectionServer"
-	ReflectionServiceInterface = "grpc.reflection.v1alpha.ServerReflection"
+	ReflectionServiceTypeName  = "ReflectionServer"
+	ReflectionServiceInterface = "dubbo.reflection.v1alpha.ReflectionServer"
```

Comment:
Maybe we should no modify this Interface since it is relevant to http path that this service would expose.

Reply:
This will conflict with grpc